server/src/main/java/io/unitycatalog/server/persist/utils/FileUtils.java

```diff
@@ -112,7 +112,7 @@ private static void deleteLocalDirectory(Path dirPath) throws IOException {
                 });
       }
     } else {
-      throw new IOException("Directory does not exist: " + dirPath);
+      LOGGER.info("Directory does not exist: {}", dirPath);
```

Comment:
why this change?

Reply:
TBH i don't think we should have ever thrown an error here. This function is used to delete managed tables/volumes. The current code throws this exception here and then catches it later [here] and logs it as an error (https://github.com/unitycatalog/unitycatalog/blob/f9ca3170d0ca6f5de7e96ab1564c72b76c5b5786/server/src/main/java/io/unitycatalog/server/persist/TableRepository.java#L304). But since we're checking for `Files.exist` here, we should just log it here rather than throw/catch/log.

Reply:
`FileUtils::deleteDirectory` should throw an exception when the dir doesn't exist. It should be up to application code to decide what to do with it. The fact that all usages right now catch-and-log exception is just happenstance. Def. undo it.